fastlane/Fastfile

```diff
@@ -7,31 +7,65 @@ end
+  { glotpress: "ko", android: "ko",    google_play: "ko-KR",  promo_config: {} },
+  { glotpress: "nl", android: "nl",    google_play: "nl-NL",  promo_config: {} },
+  { glotpress: "pl", android: "pl",    google_play: "pl-PL",  promo_config: {} },
+  { glotpress: "pt-br", android: "pt-rBR", google_play: "pt-BR",  promo_config: {} },
+  { glotpress: "ru", android: "ru",    google_play: "ru-RU",  promo_config: {} },
+  { glotpress: "sr", android: "sr",    google_play: "sr",     promo_config: {} },
+  { glotpress: "sv", android: "sv",    google_play: "sv-SE",  promo_config: {} },
+  { glotpress: "th", android: "th",    google_play: "th",     promo_config: {} },
+  { glotpress: "tr", android: "tr",    google_play: "tr-TR",  promo_config: {} },
+  { glotpress: "vi", android: "vi",    google_play: "vi",     promo_config: {} },
+  { glotpress: "zh-cn", android: "zh-rCN", google_play: "zh-CN",  promo_config: {} },
+  { glotpress: "zh-tw", android: "zh-rTW", google_play: "zh-TW",  promo_config: {} },
+  { glotpress: "az", android: "az", promo_config: false},
+  { glotpress: "el", android: "el", promo_config: false},
+  { glotpress: "es-mx", android: "es-rMX", promo_config: false},
+  { glotpress: "es-cl", android: "es-rCL", promo_config: false},
+  { glotpress: "gd", android: "gd", promo_config: false},
+  { glotpress: "hi", android: "hi", promo_config: false },
+  { glotpress: "hu", android: "hu", promo_config: false},
+  { glotpress: "nb", android: "nb", promo_config: false},
+  { glotpress: "pl", android: "pl", promo_config: false},
+  { glotpress: "th", android: "th", promo_config: false},
+  { glotpress: "uz", android: "uz", promo_config: false},
+  { glotpress: "zh-tw", android: "zh-rHK", promo_config: false},
+  { glotpress: "eu", android: "eu", promo_config: false},
+  { glotpress: "ro", android: "ro", promo_config: false},
+  { glotpress: "mk", android: "mk", promo_config: false},
+  { glotpress: "en-au", android: "en-rAU", promo_config: false},
+  { glotpress: "sr", android: "sr", promo_config: false},
+  { glotpress: "sk", android: "sk", promo_config: false},
+  { glotpress: "cy", android: "cy", promo_config: false},
+  { glotpress: "da", android: "da", promo_config: false },
+  { glotpress: "bg", android: "bg", promo_config: false },
+  { glotpress: "sq", android: "sq", promo_config: false},
+  { glotpress: "hr", android: "hr", promo_config: false},
+  { glotpress: "cs", android: "cs", promo_config: false},
+  { glotpress: "pt-br", android: "pt-rBR", promo_config: false},
+  { glotpress: "en-ca", android: "en-rCA", promo_config: false },
+  { glotpress: "ms", android: "ms", promo_config: false},
+  { glotpress: "es-ve", android: "es-rVE", promo_config: false},
+  { glotpress: "gl", android: "gl", promo_config: false},
+  { glotpress: "is", android: "is"},
+  { glotpress: "es-co", android: "es-rCO", promo_config: false },
+  { glotpress: "kmr", android: "kmr", promo_config: false},
```

Comment:
👍 

_(Hopefully we will soon be able to get rid of those ugly maps in a future PR when we do a deeper refactoring of our translation tooling and likely use https://github.com/wordpress-mobile/fastlane-locale-map/, which will make all that ugly long list way shorter and readable… but we're not there yet 😅 )_